src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -68,7 +68,12 @@ Namespace Microsoft.CodeAnalysis.Semantics
                 Case BoundKind.EventAccess
                     Return boundNode.Syntax.Parent.Kind() = SyntaxKind.AddHandlerStatement OrElse
                            boundNode.Syntax.Parent.Kind() = SyntaxKind.RemoveHandlerStatement OrElse
-                           boundNode.Syntax.Parent.Kind() = SyntaxKind.RaiseEventAccessorStatement
+                           boundNode.Syntax.Parent.Kind() = SyntaxKind.RaiseEventStatement
+                Case BoundKind.FieldAccess
+                    Return boundNode.Syntax.Parent.Kind() = SyntaxKind.RaiseEventStatement
```

Comment:
>Return boundNode.Syntax.Parent.Kind() = SyntaxKind.RaiseEventStatement [](start = 20, length = 70)

Why do we need this code and the case below? Could you elaborate with a comment?

Reply:
Not needed anymore. Will be removed when pulling from master